modular_zubbers/code/modules/customization/species/proteans/_proteans_species.dm

```diff
@@ -103,8 +104,26 @@
 	)
 	add_verb(gainer, protean_verbs)
 
+/datum/species/protean/proc/organ_reject(mob/living/source, obj/item/organ)
+	SIGNAL_HANDLER
+
+	if(isnull(owner))
```

Comment:
don't check owner, check source, it really should be guaranteed anyway, probably